external/vcm/docs/thermodynamics.rst

```diff
@@ -4,26 +4,29 @@ Thermodynamics
 Atmospheric pressure
 --------------------
 
-   .. automethod:: vcm.calc.thermo.pressure_at_midpoint
-   .. automethod:: vcm.calc.thermo.pressure_at_midpoint_log
-   .. automethod:: vcm.calc.thermo.pressure_at_interface
-   .. automethod:: vcm.calc.thermo.surface_pressure_from_delp
+   .. automethod:: vcm.calc.vertical_coordinate.pressure_at_midpoint
```

Comment:
While you are touching these docs...would you mind re targeting all these docs to top-level APIs if present?
```suggestion
   .. autofunction:: vcm.pressure_at_midpoint
```
(note that functions should be documented with `autofunction` rather than `automethod`).